image.yaml

```diff
@@ -210,8 +210,8 @@ ports:
     - value: 8443
     - value: 8778
 sources:
-    - artifact: jolokia-jvm-1.3.5.redhat-1-agent.jar
-      md5: 240381af7039461f3472b7796fe9cd4b
+    - artifact: jolokia-jvm-1.3.6.redhat-1-agent.jar
```

Comment:
This differs from the other images, e.g. https://github.com/jboss-container-images/jboss-eap-6-openshift-image/blob/eap64-dev/image.yaml#L185